pinot-segment-local/src/test/java/org/apache/pinot/segment/local/recordtransformer/RecordTransformerTest.java

```diff
@@ -178,6 +179,150 @@ public void testSanitationTransformer() {
     }
   }
 
+  @Test
+  public void testScalarOps() {
+    TableConfig tableConfig = new TableConfigBuilder(TableType.OFFLINE).setTableName("testTable").build();
+
+    // expression true, filtered
+    GenericRow genericRow = getRecord();
+    tableConfig.setIngestionConfig(
+        new IngestionConfig(null, null,
+            new FilterConfig("svInt = 123"), null, null));
```

Comment:
this shouldn't work rt? it should be svInt == 123

Reply:
In SQL, we use `=` for comparison, e.g. `SELECT * FROM myTable WHERE svInt = 123`